papers/orhan_eroglu/00_geocat.rst

```diff
@@ -93,54 +93,52 @@ on the development of data analysis and visualization functions that operate
 directly on the native grid as well as establish an active community of
 user-contributors.
 
-This paper will provide insights about the GeoCAT's software stack and
+This paper will provide insights about GeoCAT's software stack and
 current status, team scope and near-term plans, open development methodology,
-as well as ways of community involvement.
+as well as current pathways of community involvement.
 
 GeoCAT Software
 ---------------
 
-The GeoCAT team develops and maintains several open-source software tools. Before
-describing those tools, it is vital to explain in detail how the team implements the
-continuous integration and continuous delivery/deployment (CI/CD) in consistence
-for all of those tools.
+The software tools maintained by GeoCAT implement
+continuous integration and continuous delivery/deployment (CI/CD).
 
 Continuous Integration and Continuous Delivery/Deployment (CI/CD)
 =================================================================
 
-GeoCAT employs a continuous delivery model, which has been achieved through the use
-of a monthly package release cycle on package management systems and package indexes
+GeoCAT employs a continuous delivery model, with 
+a monthly package release cycle on package management systems and package indexes
 such as Conda :cite:`conda` and PyPI :cite:`conda`. This model helps the team make
 new functions available as soon as they are implemented and address potential errors
-pretty quickly. To assist this process, the team utilizes advanced  technologies
-throughout Github assets to ensure automation, unit testing and code coverage, as
+quickly. To assist this process, the team utilizes multiple tools
+throughout GitHub assets to ensure automation, unit testing and code coverage, as
 well as licensing and reproducibility. Figure :ref:`fig1badges`, for example,
 illustrates the set of badges from one of our software tools, GeoCAT's computational
 component, GeoCAT-comp, which shows the near real-time status of each CI/CD
-implementation at any time in the Github repository homepage.
+implementation at any time in the GitHub repository homepage.
 
 CI build tests of our repositories are implemented and
 automated (for pushed commits, pull requests, and daily scheduled execution) via
-Github Actions workflows :cite:`githubactions`, and the "CI" badge shows if those
-workflows are passing or failing. Similarly, the "CONDA-BUILDS" badge shows if the
+GitHub Actions workflows :cite:`githubactions`, and the "CI" badge displays the status of those workflows.
+Similarly, the "CONDA-BUILDS" badge shows if the
 conda recipe works successfully for the repository. The percent code coverage through
```

Comment:
Is there a better way to format the text for the github badges?

Reply:
Changed the format to italic uppercase.